src/authentication/background/auth-background.ts

```diff
@@ -0,0 +1,49 @@
+import { AuthService } from 'src/authentication/background/auth-service'
+import {
+    FirebaseFunctionsAuth,
+    FirebaseFunctionsSubscription,
+} from 'src/authentication/background/firebase-functions-subscription'
+import { remoteEventEmitter } from 'src/util/webextensionRPC'
+import { AuthFirebase } from 'src/authentication/background/auth-firebase'
+import { MockAuthImplementation } from 'src/authentication/background/mocks/auth-mocks'
+
+export class AuthBackground {
```

Comment:
The convention is for a background module to live in `background/index.ts`

Reply:
Changed!